garnett/serializers/base.py

```diff
@@ -0,0 +1,138 @@
+from django.core.serializers.base import Serializer as BaseSerializer
+from django.utils.encoding import is_protected_type
+from garnett.fields import TranslatedField
+
+
+def crazy(queryset):
```

Comment:
This should probably have a doc-string explaining the craziness. It should probably also not be called 'crazy' even if it is crazy.

Reply:
fixed.